suite-native/module-device-onboarding/src/screens/UninitializedDeviceLandingScreen.tsx

```diff
@@ -65,11 +64,15 @@ export const UninitializedDeviceLandingScreen = ({
 >) => {
     const hasDeviceFirmwareInstalled = useSelector(selectHasDeviceFirmwareInstalled);
     const shouldOfferUpdateFirmware = useSelector(selectShouldOfferUpdateFirmware);
-    const deviceModel = useSelector(selectDeviceModel);
+
+    // Use ref to make sure the device model doesn't change upon device disconnect.
```

Comment:
I would find this a bit more helpful if I came to refactor this at a later stage 🤔 
```suggestion
    // Use ref to make sure the device model doesn't change upon device disconnect, which would re-render the device model image
```

Reply:
Will fix.